src/main/java/org/bigml/binding/BigMLClient.java

```diff
@@ -265,7 +265,8 @@ private void initConfiguration() {
 
     private void initResources() {
         // Lets create the storage folder in it was informed
-        this.cacheManager = new CacheManager(storage);
+        //this.cacheManager = new CacheManager(storage);
+        this.cacheManager = null;
```

Comment:
Is this assignment a left-off? Or what is the reason for short-circuiting storage?

Reply:
The use of cache manager makes some thinks not to work properly. I decided to avoid the use of the cache for now and we can investigate it once all features are in place.

Reply:
Fine -- please, add some specific TODO remarks as to why the cache is making things fails and that this should be dealt with soon to reinstate the cache. Otherwise, we will miss this. Also, if you want, document it in the CHANGES or DOC, otherwise if we have some users relying on this functionality, they will have a bad day when they upgrade.